operator/helper/parser.go

```diff
@@ -122,5 +145,21 @@ func (p *ParserOperator) ProcessWith(ctx context.Context, entry *entry.Entry, pa
 	return nil
 }
 
+func (p *ParserOperator) Skip(ctx context.Context, entry *entry.Entry) (bool, error) {
+	if p.IfExpr != nil {
```

Comment:
style nit: If the condition is inverted, we could move the final return up. This would avoid having most of the code in a block, and would keep the early return condition adjacent to the check that detects it.

```
if p.IfExpr == nil {
  return false, nil
}

// evaluate match
```